### PR TITLE
style(all): Updated references to styleguide package

### PR DESCRIPTION
**Pull request by archana-s**
Styleguide is now in its own repo. Referencing to this repo from inside deck. 

/#/styleguide will still display the generated styleguide. All the CSS is minified and loaded inside spinnaker.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -48,6 +48,21 @@
...
+    autoprefixer "^8.0.0"
+    cssnano "^3.10.0"
+    gulp "^3.9.1"
+    gulp-concat "^2.6.1"
+    gulp-postcss "^7.0.1"
+    postcss-css-variables "^0.8.0"
+    postcss-export-vars "^0.2.0"
+    postcss-extend "^1.0.5"
+    postcss-import "^11.1.0"
+    postcss-style-guide "^0.14.0"
```
Comment (jrsquared) [inline yarn.lock]: any dependencies we can take out of deck now that all the styleguide stuff is in it's own repo? Maybe some of the postcss stuff?

Comment (archana-s) [reply yarn.lock]: Sorry, I forgot. I will remove all of them.

**Comment by anotherchrisberry** [comment]
@archana-s ping

File: yarn.lock (hunk 1 above)
Comment (archana-s) [reply yarn.lock]: done and done.

**Pull request merged**